app/views/assessments/viewGradesheet.html.erb

```diff
@@ -196,8 +202,8 @@ Interaction between gradesheet and annotations
 <span id="interaction_badge" class="badge" data-badge-caption="Feature Information"></span></a>
 </div>
 
-<div id="table_container" style="text-align: center">
-  <table id="grades"></table>
+<div id="table_container" style="text-align: center; width:100%; overflow: scroll">
```

Comment:
Similar to the other PR, could we change this to `overflow: auto`?

Before

![Screenshot 2023-01-14 at 23 14 09](https://user-images.githubusercontent.com/9074856/212522600-a69eac31-9660-4924-8adb-b9af44369d6d.png)

After

![Screenshot 2023-01-14 at 23 13 54](https://user-images.githubusercontent.com/9074856/212522595-ffbf4039-c45e-4c25-854c-a0c0ef192b7a.png)

Reply:
done, and moved style into stylesheet.